config/systems.php

```diff
@@ -578,4 +578,11 @@
         'manufacturer' => null,
         'active' => true,
     ],
+    102 => [
+        'name' => 'Standalones',
+        'name_full' => 'Standalones',
+        'name_short' => 'Indie',
+        'manufacturer' => '',
+        'active' => false,
+    ],
```

Comment:
This causes the tooltip to show Standalones (maybe it should be singular)

![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/04a0e01d-1960-4068-aac2-18b242ff4a7e)

and Indie is shown in the recent games list.

![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/78e68077-1151-4f7a-a2b8-8ac8718651b4)

Also, I'm not sold on 'Indie' being the short form of Standalones, but I don't have any better ideas at this time.

Reply:
Changed the short name to "N/A" and set the full name to singular "Standalone".